module-2/backend/week_one.md

```diff
@@ -1,34 +1,64 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
-
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  GET (read), POST (create), PUT (update/replace), DELETE (destroy/delete), PATCH (modify)
 2. What is Sinatra?
+  Like "baby rails" -- more lightweight version of rails
 4. What is MVC?
+  "Model-View-Controller" -- framework for how our program interact with databases, servers and the web
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  for consistency and so our program will work/be functional
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Instance variables mentioned in a route are available in in the ERB rendered from/in the same route 
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
-
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+   ```ruby
+  get '/horses' do
+     @name = "Mr. Ed"

```

Comment:
dangit! i misread this initially! to pass a local variable, you'd do something like:

get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end